### PR TITLE
Update pr mr tests

### PR DESCRIPTION
**Pull request by fredbonin**
Modified the pipeline with the latest strategy.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -29,15 +29,10 @@ jobs:
       - name: Install Node.js modules
         run: |
           npm install -g pnpm@6.2.3
-          pnpm install --unsafe-perm --frozen-lockfile
```
Comment (Brian969) [inline .github/workflows/build.yml:32]: You removed the lock file which pins the dependencies?  We need to ensure everything is pinned.

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -29,15 +29,10 @@ jobs:
...
-      - name: Fix nasty bug in CDK
-        run: |
-          # This fix is needed in order to run initial-setup/templates tests
-          # Without the fix, SynthUtils.toCloudFormation fails
-          find node_modules -name runtime-info.js -exec sed -i 's/mod.paths/(mod.paths || [])/g' {} \;
```
Comment (Brian969) [inline .github/workflows/build.yml:37]: Is this nasty bug no longer an issue?

File: .github/workflows/lint-prettier.yml [hunk 3]
```diff
@@ -32,7 +32,8 @@ jobs:
           pnpm install --frozen-lockfile
       - name: Analyze TypeScript files
         run: |
-          pnpm recursive run lint
```
Comment (Brian969) [inline .github/workflows/lint-prettier.yml:35]: Can you look at #838 - Antoine figured out how to bring down the time it takes eslint in the ci/cd pipeline down to about 6-7min - 20+minutes is insanely frustrating.  He ended up adding the .eslintrc to each project, for some reason having only 1 was making the eslint go hunting for references for a long time. Adding them to each project dropped each from 20-30 sec to 5-6 seconds each.  Is this a good approach?  How do we ensure each project has the same eslint settings?  DO we add a script to auto-addupdate these as projects are added?

File: .github/workflows/build.yml (hunk 1 above)
Comment (fredbonin) [reply .github/workflows/build.yml:32]: Fixed in the latest commit

**Review by Brian969 — APPROVED** [review]
LGTM

**Pull request merged**